docs/src/guide/parallelization.md

```diff
@@ -52,60 +52,109 @@ as a breakdown over individual routines.
-depend on both hardware and the problem
-(e.g. number of bands, ``k``-Points, FFT grid size).
-
-For the moment DFTK does not offer an automated selection mechanism
-of thread and parallelization threading and just uses the Julia defaults.
-Since these are rarely good,
-users are advised to use the timing capabilities of DFTK
-to experiment with threading
-for their particular use case before running larger calculations.
-
-### FFTW threads
-For typical small to medium-size calculations in DFTK the largest part of time is spent
-doing discrete Fourier transforms (about 80 to 90%).
-For this reason parallelising FFTs can have a large effect on the runtime
-of larger calculation in DFTK.
-Unfortunately for scaling of FFT threading for smaller problem sizes
-and large numbers of threads is not great,
-such that by default threading in FFTW is even disabled.
-
-The **recommended setting** for FFT threading with DFTK
-is therefore to only use moderate number of FFT threads,
-something like ``2`` or ``4`` and for smaller calculations
-disable FFT threading completely.
-To **enable parallelization of FFTs** (which is by default disabled),
-use
+## Options for parallelisation
+At the moment DFTK offers two ways to parallelise a calculation,
+firstly shared-memory parallelism using threading
+and secondly (possibly distributed) multiprocessing using MPI
+(via the [MPI.jl](https://github.com/JuliaParallel/MPI.jl) Julia interface).
+MPI-based parallelism is currently only over `k`-Points,
+such that it cannot be used for calculations with only a single `k`-Point.
+Otherwise combining both forms of parallelism is possible as well.
+
+The scaling of both forms of parallelism for a number of test cases
+is demonstrated in the following figure.
+These values were obtained using DFTK version 0.1.17 and Julia 1.6
+and the precise scalings will likely be different
+depending on architecture, DFTK or Julia version.
+The rough trends should, however, be similar.
+
+```@raw html
+<img src="scaling.png" width=750 />
 ```
-using FFTW
-FFTW.set_num_threads(N)
+
+The MPI-based parallelisation strategy clearly shows a superior scaling
+and should be preferred if available.
+
+
+## MPI-based parallelism
+Currently DFTK uses MPI to distribute on `k`-Points only.
+This implies that calculations with only a single `k`-Point
+cannot use make use of this.
+The recommended way to use MPI with DFTK is to
+disable all threading and run Julia with the number
+of desired MPI processes.
+For details on setting up and configuring MPI with Julia
+see the [MPI.jl documentation](https://juliaparallel.github.io/MPI.jl/stable/configuration).
+
+1. To ensure the thread setup in DFTK is consistent with the environment
+   under which DFTK runs
+   add the following to your script running the DFTK calculation:
+   ```julia
+   using DFTK
+   setup_threading()
+   ```
+   This will ensure that all threading is disabled
+   (which unfortunately is not the default).
```

Comment:
I don't know if I would call it unfortunate. What's unfortunate is that OpenBLAS sucks.

Reply:
I would not talk about threads at all in this section (except for saying don't use any) and leave it for the next one

Reply:
Ok I worked around it by adding another helper function `disable_threading`.